File: DevExpress-Demo/CS/MVVMDemo.Wpf/Modules/Commands/POCOCommandsChangeNotificationsViewModel.cs
using DevExpress.Mvvm.POCO;
using System.Windows;

namespace MVVMDemo.Commands {
    public class POCOCommandsChangeNotificationsViewModel {
        public virtual bool CanExecuteSaveCommand { get; set; }

        #region !
        protected void OnCanExecuteSaveCommandChanged() {
            this.RaiseCanExecuteChanged(x => x.Save());
        }
        #endregion

        public void Save() {
            MessageBox.Show("Action: Save");
        }
        public bool CanSave() {
            return CanExecuteSaveCommand;
        }

        protected POCOCommandsChangeNotificationsViewModel() {
            CanExecuteSaveCommand = true;
        }
    }
}

File: DevExpress-Demo/VB/MVVMDemo.Wpf/Modules/Commands/POCOCommandsChangeNotificationsViewModel.vb
Imports DevExpress.Mvvm.POCO
Imports System.Windows

Namespace MVVMDemo.Commands

    Public Class POCOCommandsChangeNotificationsViewModel

        Public Overridable Property CanExecuteSaveCommand As Boolean

#Region "!"
        Protected Sub OnCanExecuteSaveCommandChanged()
            RaiseCanExecuteChanged(Sub(x) x.Save())
        End Sub

#End Region
        Public Sub Save()
            MessageBox.Show("Action: Save")
        End Sub

        Public Function CanSave() As Boolean
            Return CanExecuteSaveCommand
        End Function

        Protected Sub New()
            CanExecuteSaveCommand = True
        End Sub
    End Class
End Namespace
